shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h

```diff
@@ -107,6 +107,13 @@ NS_ASSUME_NONNULL_BEGIN
              (void (^)(UNNotificationPresentationOptions options))completionHandler
     API_AVAILABLE(ios(10));
 
+/**
+ * Calls all plugins registered for `UNUserNotificationCenterDelegate` callbacks.
+ */
+- (void)userNotificationCenter:(UNUserNotificationCenter*)center
```

Comment:
These also need guards

Reply:
Should be `- (void)userNotificationCenter(UNUserNotificationCenter*)center API_AVAILABLE(ios(10));` I believe

Reply:
Wow I missed that you ahve this two lines below :)